src/components/Accordion/Accordion.spec.tsx

```diff
@@ -66,6 +66,22 @@ describe("Accordion Component", () => {
         cy.get("@onClickStub").should("be.calledOnce");
     });
 
+    it("opens select sections initially", () => {
+        mount(
+            <Accordion>
+                <AccordionItem header={{ children: "1" }}>
+                    <TextInput />
+                </AccordionItem>
+                <AccordionItem header={{ children: "2", active: true }}>
+                    <TextInput />
+                </AccordionItem>
+                <AccordionItem header={{ children: "3" }}>3</AccordionItem>
+            </Accordion>,
+        );
+
+        cy.get(ACCORDION_ITEM_CONTENT_ID).should("exist");
```

Comment:
Not a good test, you need to verify if the 2nd is correctly open and the rest still closed